09-echo_data.ipynb

```diff
@@ -18,10 +18,11 @@
     "from res.sequential_tasks import EchoData\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.            # we share the hidden state across successive invocations of .forward.
</pre><p>You shouldn't talk about calling the <code>.forward()</code> method. We are actually calling the model itself, which does additional stuff. If you're looking for the generic term of forward propagation, just type that in plain English. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Atcold/pytorch-Deep-Learning/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Thanks! `Changed` it to just say `successive invocations`.